src/jmclient/wallet.py

```diff
@@ -543,29 +543,20 @@ def get_key_from_addr(self, addr):
         privkey = self._get_key_from_path(path)[0]
         return privkey
 
-    def _get_addr_int_ext(self, address_type, mixdepth):
```

Comment:
In the spirit of reducing the number of function calls, I agree with this. Essentially we are changing from, e.g. (get_external_address -> get_address_int_ext -> get_external_script -> get_new_script ...) to (get_external_address -> get_new_address -> get_new_script ...). The extra level of indirection wasn't more than a binary toggle, realistically, so why not remove it, since we're already carrying the bip32 branch down the stack (so even if we want to start using branch 2,3,4, this wasn't technically needed).

Reply:
Mostly I was trying to remove redundant code. "The best part is no part." Really there are *so* many redundant methods in `wallet.py`, but I didn't want to generate a huge snarl of trivial changes by culling them all, so I settled for dropping the ones that aren't referenced elsewhere.